pkg/boot/bzimage/bzimage.go

```diff
@@ -387,6 +410,27 @@ func (h *LinuxHeader) Show() []string {
 	return s
 }
 
+// Show stringifies a LinuxParams into a []string.
+func (h *LinuxParams) Show() []string {
+	var s []string
+
+	val := reflect.ValueOf(*h)
+	for i := 0; i < val.NumField(); i++ {
+		v := val.Field(i)
+		k := reflect.ValueOf(v).Kind()
+		n := val.Type().Field(i).Name
+		switch k {
+		case reflect.Slice:
+			s = append(s, fmt.Sprintf("%s:%#02x", n, v))
```

Comment:
Slice is the same as default? How about removing it?

Reply:
ah, I'll take a look. this is really old stuff.

Reply:
Done.

`"%s:%#v"` for default branch ? -- linux params struct has map, and uint8, ... and will fall under default.